portfolio/src/main/java/com/google/sps/servlets/ChartServlet.java

```diff
@@ -0,0 +1,51 @@
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.LinkedHashMap;
+import java.util.Scanner;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that gets data from a csv and parses it to display in a chart.*/
+@WebServlet("/chart-data")
+public class ChartServlet extends HttpServlet {
+
+  // Ordered map to store the pairs of values parsed from the csv.
+  private LinkedHashMap<String, Integer> memeSearchTrend = new LinkedHashMap<>();
+
+  /**
+   * Parses csv data a single time instead of on every request.
+   * The file should be in the format date, interest level.
```

Comment:
nit: <date, interest> - I use '<>' to help make data formats more clear and separated from the rest of the sentence.

Reply:
Alright, done. Thanks!